mavros/08cd181/08cd181.bug

```diff
@@ -25,13 +25,13 @@ bug:
   detected-by: developer
   reported-by: contributor
   issue: https://github.com/mavlink/mavros/issues/161
-  time-reported: 2014-11-02 (00:00)
-  reproducibility: N/A
+  time-reported: 2014-11-02T00:00:00+00:00
```

Comment:
Is this correct? `get_issue_creation_date.py` with the issue link gives me `2014-11-02T13:28:17Z`.

Reply:
I just copied from the original format, adding 00 seconds because the schema was complaining :)  I suspect that with all the new fields in the bugzoo and time-machine sections there is never as case where we want this field to be "wrong". So what get_issue_creation_date is probably correct.  I can push this correction too, if @ChrisTimperley does not complain.

Reply:
`2014-11-02T13:28:17Z` is fine :+1: